apps/graphql/src/apps/manageMyBooking/dataloaders/Bookings.js

```diff
@@ -51,6 +53,8 @@ export const sanitizeBooking = (booking: BookingApiResult) => {
 
 const sanitizePassengers = (booking: BookingApiResult): Array<Passenger> =>
   booking.passengers.map(passenger => {
+    const passengerTitle = capitalize(passenger.title);
+    const title: PassengerTitle = (passengerTitle: any);
```

Comment:
It will be nice to get rid of this part, but for now I don't really have anything better 😄. And since we are just working with mocked data for now it doesn't make much sense to dig deeper into response types when it will be updated once/if we connect MMB to real endpoint.